webhook_server.py

```
@@ -0,0 +1,131 @@
+    """接收部署通知"""
+    signature = request.headers.get('X-Hub-Signature-256', '')
+    payload = request.get_data()
+    
+    if signature and not verify_signature(payload, signature):
+        logger.warning("Invalid webhook signature")
+        return jsonify({"status": "error", "message": "Invalid signature"}), 403
+    
+    data = request.json or {}
+    logger.info(f"Received deploy notification: {data}")
+    
+    # 在后台线程执行部署，避免阻塞 HTTP 响应
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)